clang/lib/Sema/SemaLookup.cpp

```diff
@@ -1248,6 +1248,13 @@ static DeclContext *findOuterContext(Scope *S) {
   return nullptr;
 }
 
+static bool isDependentAssignmentOperator(DeclarationName Name,
```

Comment:
a comment on this function similar to what you replied to me before woudl also be appreciated.